src/main/java/domain/Car.java

```diff
@@ -0,0 +1,45 @@
+package domain;
+
+import exception.CarException;
+import exception.ErrorMessage;
+
+public class Car {
+
+    private final int MIN_LENGTH = 1;
+    private final int MAX_LENGTH = 5;
```

Comment:
상수로 정의 했다면, static으로 선언하시죠!
```suggestion
    private static final int MIN_LENGTH = 1;
    private static final int MAX_LENGTH = 5;
```

static으로 선언하지 않은것과는 어떤차이가 있을까요?

Reply:
static으로 선언하는 이유는 클래스 간 공통으로 사용하는 변수를 만들 수 있기 때문입니다. 자동차 이름의 최소/최대 길이는 인스턴스마다 별도의 값을 가져선 안되기 때문에 상수로 정의한 부분은 static이 올바른 것 같습니다.